source/renderer/app/utils/formatters.js

```diff
@@ -260,24 +260,6 @@ export const toFixedWithoutRounding = (
 export const formattedNumber = (value: number | string, dp?: number): string =>
   new BigNumber(value).toFormat(dp);
 
-export const formattedCpuModel = (model: string): string => {
```

Comment:
[optional] Why moving `formattedCpuModel` out of `formatters.js`? 

I notice there's a  `formattedBytesToSize` that's related to profiling as well. So i wonder if `formatCpuInfo` should be moved back to formatters.js to keep similar functions grouped.